Sources/Foundation/CMakeLists.txt

```diff
@@ -163,6 +163,18 @@ set_target_properties(Foundation PROPERTIES
   Swift_MODULE_DIRECTORY ${CMAKE_BINARY_DIR}/swift
   INTERFACE_INCLUDE_DIRECTORIES ${CMAKE_BINARY_DIR}/swift)
 
+if(NOT BUILD_SHARED_LIBS)
+  add_dependencies(Foundation CoreFoundation uuid)
+# TODO(katei): Comment out after swift-frontend implementation
```

Comment:
Could you clarify what implementation are you referring to here?

Reply:
added 👍